.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## Description
+
+*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*
+
+*List which issues are fixed by this PR from the [issue database](https://github.com/CRED-CLUB/neopop-flutter). You must list at least one issue.*
```

Comment:
Change url to `https://github.com/CRED-CLUB/neopop-flutter/issues`

Reply:
fixed